Dockerfile

```diff
@@ -24,23 +24,22 @@ WORKDIR /fineract
 
 RUN ./gradlew --no-daemon -q -x rat -x compileTestJava -x test -x spotlessJavaCheck -x spotlessJava bootJar
 
-WORKDIR /fineract/target
-RUN jar -xf /fineract/fineract-provider/build/libs/fineract-provider*.jar
-
 # https://issues.apache.org/jira/browse/LEGAL-462
 # https://issues.apache.org/jira/browse/FINERACT-762
 # We include an alternative JDBC driver (which is faster, but not allowed to be default in Apache distribution)
 # allowing implementations to switch the driver used by changing start-up parameters (for both tenants and each tenant DB)
 # The commented out lines in the docker-compose.yml illustrate how to do this.
-WORKDIR /fineract/target/BOOT-INF/lib
-RUN wget -q https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.20/mysql-connector-java-8.0.20.jar
+WORKDIR /fineract/libs
+RUN wget -q https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.23/mysql-connector-java-8.0.23.jar
 
 # =========================================
 
-FROM gcr.io/distroless/java:11 as fineract
+#FROM gcr.io/distroless/java:11 as fineract
+FROM openjdk:11 AS fineract
```

Comment:
Hi - we put in quite a bit of work earlier to move from OpenJDK to distroless for smaller images and better security etc. It's a shame if we now have to undo this and go back to OpenJDK. Would the same changes not work with distroless?

Reply:
Arrgh, my mistake... that was not intended (was just testing something else... unrelated). I'll revert that immediately.

Reply:
Alright, distroless is in place again. Thanks for the pointer @ptuomola